order/buyer/receive/README.md

```diff
@@ -22,6 +22,19 @@ When choosing single delivery please continue on:
 
 {% page-ref page="single-delivery-order-response.md" %}
 
+## When working with the webhook API
+
+Use the [POST order webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint.
+
+* `eventName` contains the [order event name](https://docs.tradecloud1.com/connectors/webhook-connector/order-events)
+* `orderEvent` contains the actual order event
+
+## When working with the polling API
+
+Use the [POST poll](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/pollOrdersRoute) endpoint.
+
+* `order` contains the actual order
```

Comment:
```suggestion
* `order` contains the actual order in its current state
```

I'm a bit struggling with the wording, but perhaps it's good to mention that this will return the most recent order state (and eg. not the state after the first event that may have happened)